CHANGELOG.md

```diff
@@ -49,9 +51,34 @@ up, but since the state had been overwritten, it couldn't and the controller wou
 
 The router now uses a UUID as well so there shouldn't be any more orphaned terminators. 
 
+## host.v1/host.v2 change
+
+The host.v2 config type was mostly a collection of host.v1 instances. However, there was a one small difference.
+The host.v1 type had `connectTimeoutSeconds` in `listenOptions`, where `host.v2` had `connectTimeout`, defined
+as a duration. 
+
+To bring them into alignment, both attributes are now supported in both config types. However, `connectTimeout`
+will take precedence if both are present. `connectTimeout` is not yet supported by c-based tunnelers. Once it 
```

Comment:
c-based tunnelers or is this "every non-go-based sdk"?

Reply:
Every non-go-based-sdk. I updated the changelog to reflect this.